test/SaeTest.php

```diff
@@ -141,12 +96,10 @@ public function testCanPostValidData()
     public function testCannotPostInvalidData()
     {
         $this->expectExceptionMessage(
-            '{"valid":false,"errors":[{"property":"id"' .
-            ',"pointer":"\/id","message":"The property id is required","' .
-            'constraint":"required","context":1},{"property":"name","pointer":' .
-            '"\/name","message":"The property name is required","constraint":' .
-            '"required","context":1},{"property":"price","pointer":"\/price",' .
-            '"message":"The property price is required","constraint":"required","context":1}]}'
+            '{"valid":false,"errors":['.
```

Comment:
@fmizzell why are the pointers all empty with the new error logic?

Reply:
@dafeder good question. That is what is given to us by the library. Should we explore nested errors, as the pointer might be more relevant in those cases?